doc/releases/changelog-dev.md

```diff
@@ -23,7 +23,11 @@
   >>> qml.CNOT((0,1))
   CNOT(wires=[0, 1])
   ```
+  
+* Instead of checking types, objects are processed in `QuantumTape`'s based on a new `_queue_category` property.
+  [(#2408)](https://github.com/PennyLaneAI/pennylane/pull/2408)
```

Comment:
Is this a good place to add transparency? I.e., explain that we are moving away from class inheritance being correlated with the queues? 

At least I'd suggest adding the info that this is not a new stable design, but a temporary fix?